planck-cljs/src/planck/io.cljs

```diff
@@ -352,6 +352,33 @@
   :args (s/cat :dir (s/or :string string? :file file?))
   :ret boolean?)
 
+(defn ^boolean exists?
```

Comment:
This `^boolean` type hint (and others) should not be necessary, owing to [function return type inference](https://clojurescript.org/news/2018-11-02-release#_function_return_type_inference).

The existing type hint on `directory?` is necessary because its implementation calls upon a JavaScript function of unknown return type.